org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/FoldingRangeHandler.java

```diff
@@ -93,6 +93,8 @@ private void computeFoldingRanges(List<FoldingRange> foldingRanges, ITypeRoot un
 
 			int start = shift;
 			int token = 0;
+			int startImport = -1;
+			int endImport = -1;

```

Comment:
It might make sense to rename these to something like `startImportForClassFile` or `classFileImportStart`. Basically something to make it clear that we only use this when dealing with a class file.